frontend/src/employee-mobile-frontend/components/attendances/actions/MarkAbsentBeforehand.tsx

```diff
@@ -209,3 +351,35 @@ const TimeInputWrapper = styled.span`
 const Separator = styled.span`
   padding-bottom: 5px;
 `
+
+function groupAbsencesByDateRange(absences: Success<Absence[]>): LocalDate[][] {
```

Comment:
`Success` wrapper seems weird here, just unpack it on caller side already. Shouldn't this return `FiniteDateRange[]` instead of `LocalDate[][]`?

Reply:
Actually this function does not need to be absence specific at all but
`function groupDatesToRanges(dates: LocalDate[]): FiniteDateRange[]` 
which then should be unit-tested.

Reply:
Didn't realise (or I had forgotten that we had FiniteDateRange in the frontend as well. 👍